.github/workflows/ruby.yml

```diff
@@ -8,15 +8,15 @@ on:
 
 jobs:
   test:
-    runs-on: "ubuntu-latest"
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.6, 2.7, head]
+        ruby: ["2.6", "2.7", "3.0", head]
```

Comment:
[note] Quotes are necessary for numbers. See <https://github.com/actions/runner/issues/849>.